bmplib/spectral/spectrum_wwind.py

```diff
@@ -0,0 +1,63 @@
+#compute FFT of given array
+
+import numpy as np
+from scipy.signal import blackman,bartlett,hanning,hamming
```

Comment:
```suggestion
import numpy as np

from scipy.signal import blackman,bartlett,hanning,hamming
```

It's good practice to split direct import and from-import into separate groups.